app/controllers/referee_interface/reference_controller.rb

```diff
@@ -0,0 +1,47 @@
+module RefereeInterface
+  class ReferenceController < ActionController::Base
+    layout 'application'
```

Comment:
We'll need to add some things here that the other controllers have as well:

- `include LogQueryParams`
- An implementation of `before_action :add_identity_to_log` so that we know in the logs & Sentry who triggered a log or error

See the candidate bit for an example:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/d4010b0622ac6d86239210ca8197b72acbcc941a/app/controllers/candidate_interface/candidate_interface_controller.rb#L3-L7

Reply:
good point! 
I think we need all of them but `before_action :authenticate_candidate! ` since in this case, the user is the referee

Reply:
For the  `before_action :authenticate_referee!`
I am gonna use the same password as `ENV.fetch('BASIC_AUTH_PASSWORD'))`

I suppose it makes sense since who will test this in staging/production is the same user that creates the application form.

Reply:
I am not sure the data to log with `add_identity_to_log`.
Probably just the `id` of the `referee`.  (like the support controllers, candidate controller, ...)